conf/bootstrap.coffee

```diff
@@ -1,31 +1,30 @@
 
 module.exports =
-  '@nikitajs/core/lib/log/cli':
+  cli:
     pad: host: 20, header: 60
-  '@nikitajs/core/lib/log/md':
```

Comment:
you don't want to remove this

Reply:
Similar to the above command:

I put the logging process into `run.coffee`.

Reply:
No, it must work, if it don't, we will re-implement it in Nikita